src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingCook.java

```diff
@@ -87,6 +93,22 @@ public boolean canWorkDuringTheRain()
         return true;
     }
 
+    @Override
+    public int buildingRequiresCertainAmountOfItem(final ItemStack stack, final List<ItemStorage> localAlreadyKept, final boolean inventory)
+    {
+        if (stack.isEmpty())
+        {
+            return 0;
+        }
+
+        if (ISFOOD.test(stack) || getAllowedFuel().stream().anyMatch(fuelStack -> fuelStack.isItemEqual(stack)))
+        {
+            return 0;
```

Comment:
dont we need to return more than 0 if we want to keep it?

Reply:
The return quantity is the quantity the dman can gather

Reply:
ah I see the function just does the opposite of what the name implies :D